utils/src/main/scala/za/co/absa/enceladus/utils/general/ProjectMetadata.scala

```diff
@@ -15,8 +15,10 @@
 
 package za.co.absa.enceladus.utils.general
 
-object ProjectMetadata {
-
-  val enceladusVersion: String = getClass.getPackage.getImplementationVersion
+trait ProjectMetadata {
```

Comment:
Please add a comment, why the trait exists - that the object in some cases does not work (_Spring_)

Reply:
No relevant anymore.